scvid/data/dadc_dataset.py

```diff
@@ -14,11 +16,161 @@ def __init__(self, dadc: DistributedAnnDataCollection) -> None:
     def __len__(self) -> int:
         return len(self.dadc)
 
-    def __getitem__(self, idx: int) -> Dict[str, Tensor]:
+    def __getitem__(self, idx: int) -> Dict[str, np.ndarray]:
         """Return gene counts for a cell at idx."""
```

Comment:
`gene counts` -> `feature counts`. Also, mention that if the data is in sparse repr, it will be made dense.